features/util.py

```diff
@@ -231,19 +233,34 @@ def build_debs(series: str, cache_source: bool = False) -> List[str]:
 
     curr_dir = os.getcwd()
     os.chdir(SBUILD_DIR)
-    logging.info("--- Running sbuild")
+
+    sbuild_cmd = [
+        "sbuild",
+        "--no-run-lintian",
+        "--resolve-alternatives",
+        "--no-clean-source",
+        "--arch",
+        "amd64",
+        "-d",
+        series,
+        SOURCE_PR_UNTAR_DIR,
+    ]
+    if chroot is not None:
```

Comment:
I think we can also log which chroot is being used

Reply:
Indeed! I changed the log call below to include the entire sbuild command, which includes the chroot flag if present